.github/workflows/migrator.yml

```diff
@@ -433,6 +466,6 @@ jobs:
               issue_number: pr_number,
               body: `**:x: Error :x:**
 
-            An error occurred while migrating and syncing the PR. Please check the action logs for more details.`,
+            An error occurred while migrating or syncing the PR. Pinging @PyMAPDL-Maintainers for assistance.`,
```

Comment:
Shouldn't the team be in lower caps? ``@ansys/pymapdl-maintainers`` is the actual team in fact